core/ir/disassemble_shared.c

```diff
@@ -1035,7 +1035,9 @@ instr_disassemble_opnds_noimplicit(char *buf, size_t bufsz, size_t *sofar DR_PAR
                          */
```

Comment:
Is there an issue# that this bug fixes? Either way, would be nice to note the goal (cleanup/bug-fix) in the PR description.